source/agora/node/Validator.d

```diff
@@ -438,11 +438,19 @@ public class Validator : FullNode, API
 
     protected override string acceptHeader (BlockHeader header) @safe
     {
-        if (auto err = super.acceptHeader(header))
```

Comment:
`FullNode.acceptHeader` also invoked Ledger signature update, but we're missing it right now.

Reply:
It will still update the signatures when `selfSignBlock` is called.

Reply:
I might be missing something but; couldn't reach to `this.storage.updateBlockSig` that we had from new changeset

Reply:
Sorry I see what you mean now. I will add the `this.ledger.updateBlockMultiSig(header);` in `Validator`.

Reply:
Pushed an update.